tags/tags.toml

```
@@ -895,3 +895,12 @@ content = """
 + embed.setFooter({ text: 'This is an example text', iconURL: 'https://example.com/icon.png' })
 ```
 """
+
+[interaction-acknowledged]
```

```suggestion
[already-acknowledged]
```
already-acknowledged is the essence of the error